core/src/components/alert/alert.scss

```diff
@@ -91,10 +91,22 @@
   overscroll-behavior-y: contain;
 }
 
-.alert-checkbox-group::-webkit-scrollbar,
-.alert-radio-group::-webkit-scrollbar,
-.alert-message::-webkit-scrollbar {
-  display: none;
+// Scrollbars on mobile devices will be hidden.
+// Users can still scroll the content by swiping.
+// If a user has a pointing device, such as a
+// mouse or trackpad, then scrollbars will be
+// visible. This allows users to scroll the
+// content with their pointing device.
+// Otherwise, the user would have to use the
+// keyboard to navigate through the options.
+// This may not be intuitive for users who
+// are not familiar with keyboard navigation.
+@media (any-pointer: none) {
```

Comment:
Do you mean to do `any-pointer: coarse` instead? `any-pointer: none` means that this CSS will only apply when there is no pointing device available. A `coarse` pointer has limited accuracy, so that will apply on touch screens.

With this change as-is the scrollbars show up on mobile devices.

Reply:
My thought was that when no pointers are present then don't show the scrollbars. If no pointers are available then it's a mobile device. However, I just realized that touching the screen will count as a pointer. I had to dig a bit more to get a better description of coarse. I agree with changing it to `coarse`.